core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -2710,7 +2710,67 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     assert(countSubmittedMapStageAttempts() === 2)
   }
 
-  test("SPARK-23207: retry all the succeeding stages when the map stage is indeterminate") {
+  test("SPARK-25341: abort stage while using old fetch protocol") {
+    // reset the test context with using old fetch protocol
+    afterEach()
```

Comment:
do we need to call this? It should already be called when the previous test is finished and right before running this test.

Reply:
```
It should already be called when the previous test is finished and right before running this test.
```
Yeah, that's right but we still need to call this because the spark context is created in `beforeEach`, here we call `afterEach` is to stop the context which starts by the latest `beforeEach`.